scripts/README.md

```diff
@@ -6,6 +6,14 @@ This directory contains scripts to help with ggshield development.
 
 Build .pyz, .deb and .rpm packages.
 
+## action-entrypoint-generator/action-entrypoint-generator
+
+We have two GitHub actions: one to run `secret scan` and another to run `iac scan`. They are defined in `actions/secret` and `actions/iac`. These two actions use the common Dockerfile from the root of the repository, which provides a way to run GGShield inside a Docker container.
+
+To test that changes made inside pull requests do not cause problems with our GitHub actions, we have a separate set of actions: `actions-unstable/secret` and `actions-unstable/iac`. The difference between these actions and those defined in `actions`, is that if the `GITGUARDIAN_GGSHIELD_REF` environment variable is set, then they install GGShield from this ref, so that we test the code from the pull request and not the code currently in the `main` branch.
```

Comment:
We do not mention the base image being `unstable` instead of `latest` ?

Reply:
Good point, fixed.